### ISSUE TITLE
OctoPrint 1.9.0 Compatibility

### ISSUE THREAD

**Issue by jneilliii** [jneilliii/OctoPrint-WebcamIframe#21]
```
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:			$('#webcam_container').replaceWith('<iframe id="webcam_container" src="about:blank" width="588" height="330" style="border: none;" allow="autoplay *; fullscreen *"></iframe>');
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:				var iframeTimeout = self.settingsViewModel.webcam_streamTimeout();
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:					if ($('#webcam_container').attr("src") != self.settingsViewModel.webcam_streamUrl()) {
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:						$('#webcam_container').attr("src", self.settingsViewModel.webcam_streamUrl());
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:						$('#webcam_container').attr("src", "about:blank");
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:					var dashboard_webcam = $('#plugin_dashboard_webcam_container');
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:					if (dashboard_webcam.length == 0) {  // We have to do it here, because dashboard_webcam_image object not exists yet when onAllBound triggered
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:						dashboard_webcam = $('<iframe id="plugin_dashboard_webcam_container" src="about:blank" width="100%" height="100%" style="border: none;" allow="autoplay *; fullscreen *"></iframe>');
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:						$("#dashboard_webcam_image").replaceWith(dashboard_webcam);
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:					if (dashboard_webcam.attr("src") != self.settingsViewModel.webcam_streamUrl()) {
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:						dashboard_webcam.attr("src", self.settingsViewModel.webcam_streamUrl());
webcam_iframe/OctoPrint-WebcamIframe-master/octoprint_webcam_iframe/static/js/webcam_iframe.js:						$('#plugin_dashboard_webcam_container').attr("src", "about:blank");
```

**Comment by automiketic** [comment]
Any plans to add compatibility? If I run v1.9 without the new camera stack will this plugin work? I use a cam that provides a stream over WebRTC that seems like the perfect solution for :)

**Comment by jneilliii** [comment]
I was going to fix it, but completely forgot. I'll try to remember to take a look this weekend.

**Comment by davedamon** [comment]
Any update on this?

**Comment by jneilliii** [comment]
Actually on my list to look at the upcoming week on my vacation.

**Comment by jneilliii** [comment]
resolved in just released version 0.0.6 of the plugin hopefully.
